android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -387,20 +401,33 @@ public ThemedReactContext getThemedReactContext() {
   }
 
   protected class RNCWebViewBridge {
+        private String TAG = "RNCWebViewBridge";
         RNCWebView mWebView;
+        String injectedObject;
```

Comment:
ultra-nit: I wonder if it would be worth calling this `injectedObjectJson`? It seems potentially confusing that it's called an "object" but is a string...

Reply:
Good idea.